app/components/forms/wizard/basic-details-step.js

```diff
@@ -187,6 +187,10 @@ export default Component.extend(FormMixin, {
     return orderBy(licenses, 'name');
   }),
 
+  copyright: computed('data.event.copyright', function() {
```

Comment:
While creating `data.event.copyright` will be undefined and when we bind it to ui-dropdown it will throw an error, so I have added it so store the copyright info and will save/update when the form gets submitted.